src/routes.ts

```diff
@@ -238,18 +236,11 @@ export default [
     }
   },
   DigitalCredentialRoutes,
-  // {
-  //   path: '/about',
-  //   name: 'about',
-  //   // route level code-splitting
-  //   // this generates a separate chunk (about.[hash].js) for this route
-  //   // which is lazy-loaded when the route is visited.
-  //   // ref: https://cli.vuejs.org/guide/html-and-static-assets.html#prefetch
-  //   component: () => import(/* webpackChunkName: "about" */ './views/About.vue')
-  // },
-  {
-    // default/fallback route
-    path: '*',
-    redirect: '/'
+  {
+    // Return to new dashboard if we have a businessID, but no actual filing info
+    path: '/:businessId/',
```

Comment:
Does this ever get used? I ask because the business id stripped out of the route (VUE_ROUTER_BASE) in set-base-route-and-business-id.ts. In other words, the router (that uses this file) should only ever see what's after the business id, ex: "standalone-directors?filingId=0".

Meanwhile, we should still have a default/fallback route to handle such cases as "https://dev.business.bcregistry.gov.bc.ca/CP1049523/not-a-valid-filing".

I know you've investigated this a bunch so let me know what you found.